src/tests/signupTest.js

```diff
@@ -0,0 +1,156 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Test on user signup:', () => {
+  // New user account able to be created
+  it('It should create a new user', (done) => {
+    const newUser = {
+      firstName: 'Gustave',
+      lastName: 'Harintwali',
```

Comment:
Also, move generated objects/data to a separate file and import them inside the test instead.